examples/seismic/acoustic/operators.py

```diff
@@ -90,13 +118,20 @@ def ForwardOperator(model, geometry, space_order=4,
 def AdjointOperator(model, geometry, space_order=4,
                     kernel='OT2', **kwargs):
     """
-    Constructor method for the adjoint modelling operator in an acoustic media
-
-    :param model: :class:`Model` object containing the physical parameters
-    :param source: :class:`PointData` object containing the source geometry
-    :param receiver: :class:`PointData` object containing the acquisition geometry
-    :param time_order: Time discretization order
-    :param space_order: Space discretization order
+    Constructor method for the adjoint modelling operator in an acoustic media.
+
+    Parameters
```

Comment:
this docstring is also obsolete. We should have spotted it much earlier.

Reply:
This is what it does, it returns the adjoint operator, not sure why "obsolete" unles you're talking about the Parameters, then as above copy-paste Wavesolver

Reply:
@georgebisbas do you see what we're talking about here?

Reply:
Let me see it after the meeting, or discuss it there...